### PR TITLE
Bug fix zth

### PR DESCRIPTION
**Pull request by zachholcomb**
- took away merchant delete and edit privileges from regular users or visitors
 - took away creating new merchant stores from every user except admins
 - fix ordered list bug in items index 
 - update tests to match new restrictions
 - [x] All your base are belong to us!

### CODE REVIEW

File: spec/features/merchants/new_spec.rb [hunk 1]
```diff
@@ -3,6 +3,9 @@
 RSpec.describe 'merchant new page', type: :feature do
   describe 'As a user' do
```
Comment (jrsewell400) [inline spec/features/merchants/new_spec.rb:4]: Should this say, 'As an Admin'?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -28,8 +28,8 @@
         <% elsif current_user.admin? %>
           <%= link_to "Admin Dashboard", "/admin/dashboard" %>
           <%= link_to "See all Users", "/admin/users" %>
-        <span>Logged in as <%= current_user.name %></span>
         <% end %>
+        <span>Logged in as <%= current_user.name %></span>
```
Comment (jrsewell400) [inline app/views/layouts/application.html.erb]: Good catch!

File: app/views/items/index.html.erb [hunk 3]
```diff
@@ -26,18 +26,14 @@
...
     <% end %>
   </section>
   <section id="least-popular">
     <h3>Least Popular Items</h3>
-    <% @items.least_purchased.each do |item| %>
-    <ol> 
-      <li><%= item.name%>: <%= item.quantity_purchased %></li>
-    </ol>
-  <% end %>
+    <% @items.least_purchased.each_with_index do |item, index| %>
```
Comment (jrsewell400) [inline app/views/items/index.html.erb:35]: Nice use of each_with_index

**Comment by elguapogordo** [comment]
alt text

**Comment by elguapogordo** [comment]
there were a couple of `.created_at` sections that don't have `.to_date` under /admin section, as well

**Review by elguapogordo — APPROVED** [review]

**Review by elguapogordo — APPROVED** [review]
alt text

**Comment by jrsewell400** [comment]
Everything is good to go, mergingin!

**Pull request merged**